charts/single-node-values.yml

```diff
@@ -0,0 +1,24 @@
+ingress:
+  enabled: true
+  hosts:
+    - ""
+cloudserver-front:
+  orbit:
+    enabled: true
```

Comment:
Shouldn't this become the default in the `cloudserver-front` Chart, because it's basically broken without this right now? Maybe the option should even be removed until things are fixed again. WDYT @vrancurel

Reply:
until we have a clear view it is not bad to be explicit